fleetspeak/src/e2etesting/e2etest.sh

```diff
@@ -3,4 +3,5 @@ cd $(dirname $0)/../../../
 python3 -m venv /tmp/.venv/FLEETSPEAK
 /tmp/.venv/FLEETSPEAK/bin/pip install absl-py protobuf fleetspeak > /dev/null
```

Comment:
We should need absl-py anymore. Do we still need protobuf?

Reply:
Right, we don't need either of them now.